composer.json

```diff
@@ -6,16 +6,19 @@
     "type": "project",
     "require": {
         "php": ">=7.1.3",
+        "ext-json": "*",
         "barryvdh/laravel-dompdf": "^0.8.2",
+        "binarytorch/larecipe": "^1.2",
         "doctrine/dbal": "^2.5",
         "fideloper/proxy": "~4.0",
-        "gloudemans/shoppingcart": "^2.5",
+        "gloudemans/shoppingcart": "dev-master",
```

Comment:
We stick to a specific version because we know our code is working on that version. We do not want to use new codes on the latest package since it might break some of our codes.

Reply:
It's only temporary, I didn't make anything fancy just need to update laravel version in composer.json file